packages/typespec-azure-resource-manager/lib/common-types/managed-identity.tsp

```diff
@@ -76,6 +89,19 @@ alias ManagedIdentityType = ManagedServiceIdentityType;
 /**
  * A managed identity assigned by the user.
  */
+@armCommonDefinition(
```

Comment:
same here, was that just missed and it wasn't replacing with $ref?

Reply:
Same reason.  Although here, I don't know if anyone references this property directly, but we do want to generate this into common-types